docs/data/base/getting-started/overview/overview.md

```diff
@@ -0,0 +1,30 @@
+# MUI Base - Overview
+
+<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks.</p>
+
+## Introduction
+
+MUI Base is a library of unstyled React UI components.
+It features the same robust engineering as Material UI, but without implementing Material Design.
+
+It includes prebuilt components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
+
+With MUI Base, you can rapidly build on top of our foundational components using any styling solution you choose—no need to override any default style engines or settings.
+
+## Advantages of MUI Base
+
+- **Ship faster:** MUI Base gives you the foundational building blocks you need to assemble a sleek and sophisticated user interface in a fraction of the time that it would take to do it all from scratch.
+
+- **You own the CSS:** unlike Material UI, which uses Emotion as a default style engine, MUI Base has no built-in styling solution.
+  This means you have complete control over your app's CSS.
+
+- **Accessibility:** MUI Base components are built with accessibility in mind.
+  We do our best to make all components screen reader-friendly, and offer suggestions for optimizing accessibility throughout our documentation.
+
+### MUI Base vs. Material UI
+
+MUI Base features many of the same components as Material UI, but _without the Material Design implementation_.
+
+You could think of MUI Base as the "skeletal" version of Material UI—foundational components with no default theme or built-in style engine.
+
+This makes Base a great choice if you need complete control over how your app's CSS is implemented.
```

Comment:
I would add in the end some thing like: "We plan to use the base components/hooks inside the Material UI components in the upcoming versions." 

This should make developers confident that the quality of the components & hooks will always be top notch :)

Reply:
Good idea!